packages/vitest/src/node/core.ts

```diff
@@ -42,6 +42,7 @@ export class Vitest {
   filenamePattern?: string
   runningPromise?: Promise<void>
   closingPromise?: Promise<void>
+  processTimeoutCauses: string[] = []
```

Comment:
Is there any better place than this `ctx.processTimeoutCauses` to store these errors? I thought about `ctx.state.processTimeoutCauses` but I wasn't sure that was right either.

Reply:
State is probably better